tempesta_fw/http_parser.c

```diff
@@ -2717,10 +2667,16 @@ tfw_http_parse_resp(void *resp_data, unsigned char *data, size_t len)
 
 	__FSM_START(parser->state) {
 
+	/* ----------------    EOL    ---------------- */
+
+	#define TFW_HTTP_RESP_EOL
+	#include "http_parser_eol.h"
+	#undef TFW_HTTP_RESP_EOL
+
```

Comment:
Defining a macro reduces readability as it needs special formatting with back-slashes... So, in that case I decide not to use macro but move whole the EOL-logic into the separate file. I think that all the http-parser code needs to be moved into the separate directory, also.